build/common.legacy.props

```diff
@@ -12,4 +12,10 @@
 
   <!-- Additional props -->
   <Import Project="common.project.props"/>
+
+  <PropertyGroup>
+    <!-- Some legacy projects reference assemblies in GAC;AssemblyFolders. -->
+    <!-- Reset AssemblySearchPaths to empty to let Microsoft.Common.targets pick the legacy defaults -->
+    <AssemblySearchPaths />
+  </PropertyGroup>
```

Comment:
We were seeing issues and the msbuild team suggested to do this... Here is the commit description they gave us:

> Microsoft.Build.Runtime package has an mscorlib.dll content file that
ends up winning over mscorlib.dll in .NET Framework targeting pack
mscorlib.dll if CandidateAssemblyFiles are searched.
Legacy project search paths are left unchanaged because they were not
impacted by this and some of the rely on resolution in GAC or
AssemblyFolders.